src/modules/infobox/components/FeatureElevationProfilePlot.vue

```diff
@@ -5,6 +5,18 @@
         @mouseenter="startPositionTracking"
         @mouseleave="stopPositionTracking"
     >
+        <div v-if="elevationProfile.segmentsCount > 1" class="d-flex gap-1 segment-container">
+            <button
+                v-for="(_, index) in elevationProfile.segments"
+                :key="index"
+                class="btn btn-light text-nowrap"
+                :class="{ 'btn-dark': index === elevationProfile.activeSegmentIndex }"
```

Comment:
I know it's red, and usually red is more for "danger", but I think it would be better to use `btn-primary` anyway. The black button is only used once in the app, for the open/close menu button.